docs/upgrade/0.4.x_to_0.5.rst

```diff
@@ -0,0 +1,76 @@
+Upgrade from 0.4.x to 0.5.x
+===========================
+
+Beginning with SecureDrop 0.5, the source and journalist interfaces
+are localized. After an unattended upgrade of the application server,
+these translations are available but are not activated by default.
+
+.. note::
+   See the :doc:`installation documenation <../install>` for a list
+   of supported languages.
+
+The steps below should be performed on **the Admin Workstation**
+associated with your SecureDrop instance.
+
+Pull the latest release
+-----------------------
+
+Open a **Terminal** and navigate to your SecureDrop directory.
+
+.. code:: sh
+
+   cd ~/Persistent/securedrop
+
+Stash your local configuration, fetch the latest code, and verify the tag for the
+latest release (0.5):
+
+.. code:: sh
+
+   git stash save "site specific configs"
```

Comment:
Admins shouldn't need to do the stash and pop at the end (that was only needed in 0.3.x before we had `site-specific` - now config is not stored in a file under version control). We can instead add a note to the top of this doc that tells admins to first follow the instructions in the 0.3->0.4 migration document if they had _not_ yet updated their admin workstation to 0.4. Else they can just proceed with fetching the new release branch and verifying the tag.

Reply:
Ah, I did not realize that and was curious about this recommendation .I thought some people override files found in the repository for some reason. done